### DISCUSSION TITLE
Aiuto Installazione

### DISCUSSION THREAD

**Discussion by ISConvertini2022** [iisgiua/giuaschool#316] [Richieste di aiuto]
Salve,
Vorremmo iniziare ad utilizzare il Registro Guia, poiché OpenSource, ma stiamo incontrando difficoltà nell'installazione.
Possediamo ovviamente un Hosting AlterVista, ma i tempi di caricamento dei dati su di esso sono risultati estremamente lunghi.
Per evitare problemi, e risparmiare tempo, siamo a chiederVi se è possibile che siate voi ad installare il Registro (previa fornitura delle nostre Credenziali d'Accesso ad AlterVista.

Grazie.
Cordiali Saluti

**Comment by trinko** [comment]
Ho eliminato l'email dal testo per la privacy e risposto privatamente.

**Comment by ISConvertini2022** [comment]
Grazie, 
se qualcuno degli utenti è in grado di fornirci un aiuto o assistenza, saremmo infinitamente grati. Nel caso aveste la possibilità, rispondete alla discussione o contattateci in privato.
Grazie
